Python/samples/house-energy/machine_teacher.ink

```diff
@@ -53,10 +54,12 @@ graph (input: ObservableState): SimAction {
                     C: 0.3,
                     Qhvac: 9,
                     Tin_initial: number<18 .. 30>,
-                    schedule_index: 2,
-                    number_of_days: 1,
-                    timestep: 5,
-                    max_iterations: MaxIterations
+                    Tout_median: number<18 .. 25>, # C,
+                    Tout_amplitude: number<3 .. 5>,
+                    Tset_start: Number.Int8<20 .. 22>,
+                    Tset_stop: 25,
+                    Tset_transition: Number.Int8<8 .. 12>,
```

Comment:
Tset_transition for me is set to 100..124. Do you calcculate things in minutes or something?